modules/gdpr_tasks/src/Traversal/RightToAccessEntityTraversal.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace Drupal\gdpr_tasks\Traversal;
+
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Entity\FieldableEntityInterface;
+use Drupal\Core\Field\FieldDefinitionInterface;
+use Drupal\gdpr_fields\Entity\GdprFieldConfigEntity;
+use Drupal\gdpr_fields\EntityTraversal;
+
+/**
+ * Entity traversal for performing Right to Access requests.
+ *
+ * @package Drupal\gdpr_tasks
+ */
+class RightToAccessEntityTraversal extends EntityTraversal {
+  private $assets;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function traverse(EntityInterface $entity) {
+    $this->assets = [];
+    $results = parent::traverse($entity);
+    $results['_assets'] = $this->assets;
+    return $results;
```

Comment:
If we are using `$this->assets`, can we clear it at the end as well?

Reply:
Done